packages/block-library/src/embed/embed-placeholder.js

```diff
@@ -24,7 +24,7 @@ const EmbedPlaceholder = ( props ) => {
 				</Button>
 				{ cannotEmbed &&
 					<p className="components-placeholder__error">
-						{ __( 'Sorry, we could not embed that content.' ) }<br />
+						{ __( 'Sorry, this content could not be embed.' ) }<br />
```

Comment:
I don't really think this makes sense as is. 

```suggestion
						{ __( 'Sorry, this content could not be embedded.' ) }<br />
```

Reply:
I've updated. Thanks for show me this mistake. I'm not native/fluent english speaker >.<

Reply:
No worries, thanks for updating! :)

It looks great to me now.